ports/openblas/portfile.cmake

```
@@ -16,9 +16,11 @@ if(NOT VCPKG_TARGET_ARCHITECTURE STREQUAL "x64")
     message(FATAL_ERROR "openblas can only be built for x64 currently")
 endif()
 
-if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-    message("openblas currenly only supports dynamic library linkage")
-    set(VCPKG_LIBRARY_LINKAGE "dynamic")
+if(VCPKG_CMAKE_SYSTEM_NAME  STREQUAL "WindowsStore")
```

This check will only trigger on UWP builds, not normal windows builds (as is currently required to build `OpenBLAS` successfully). I would suggest the following change which matches the `OpenBLAS` restriction

```suggestion
if(MSVC)
```